Source/loadsave.cpp

```diff
@@ -1738,7 +1738,10 @@ static void SavePortal(SaveHelper *file, int i)
 void SaveHeroItems(PlayerStruct *pPlayer)
 {
 	size_t items = NUM_INVLOC + NUM_INV_GRID_ELEM + MAXBELTITEMS;
-	SaveHelper file("heroitems", items * sizeof(ItemStruct));
+	int itemSize = 368;
+	if (gbIsHellfire)
+		itemSize + 4;

```

Comment:
Not too relevant to the change itself, but can we agree to always leave a blank line after nested statements like this as a convention?

It makes the code more readable and avoids the situation of inadvertently adding a "second" statement to a scopeless `if` thinking it would become part of the branch execution.

Reply:
Sure can, I personally think it should be mandatory when doing stuff like this.

Reply:
All formatting should be configured in `.clang-format` and done automatically, for the most part.

Reply:
Just to make it clear as I'm not sure my request was understood:

I'm proposing we do:
```cpp
if (x)
    doSomething();

doSomethingElse();
```

Instead of 
```cpp
if (x)
    doSomething();
doSomethingElse();
```

Reply:
I was by me. Can you figure out how to configure it in .clang-fomrat?

Reply:
> inadvertently adding a "second" statement

I'm not sure this is possible. The formatter will out-dent the block. If you leave it indented, the compiler or `clang-tidy` will tell you too.

To be clear, I'm against any conventions that are not taken care of automatically.
I work with *many* codebases and can't remember all the conventions. That's what tooling is for.

If clang-format provides such an option, that's fine by me https://clang.llvm.org/docs/ClangFormatStyleOptions.html

Reply:
> To be clear, I'm against any conventions that are not taken care of automatically.
I work with many codebases and can't remember all the conventions. That's what tooling is for.

I'm with you. Not sure if my suggestion sounded like I wanted to do it manually only?

> If clang-format provides such an option, that's fine by me https://clang.llvm.org/docs/ClangFormatStyleOptions.html

I'll take a look at that, thanks for the link.

Reply:
> Not sure if my suggestion sounded like I wanted to do it manually only?

Yeah I think I thought this because you used the word convention

> I'll take a look at that, thanks for the link.

If there isn't an option and you're willing, clang-format supports plugins

Reply:
agreed, automated or free for all